account_chart_update/wizard/wizard_matching.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019 Tecnativa - Ernesto Tejeda
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class WizardMatching(models.TransientModel):
```

Comment:
Don't put this in extra file. They are all children of the same main wizard.

Reply:
done